targets/unity-v2/make.js

```diff
@@ -389,12 +436,11 @@ function getRequestActions(tabbing, apiCall) {
             tabbing + "if (authType == AuthType.None && !string.IsNullOrEmpty(PlayFabSettings.DeveloperSecretKey))\n" +
             tabbing + "    authType = AuthType.DevSecretKey;\n" +
             "#endif\n";
-
```

Comment:
Let's keep this empty line.
GetEntityToken is a single API method that serves as a very special case, where all the ones below are standard api authentication handling.